docs/prover/cli/options.md

```diff
@@ -726,19 +726,26 @@ reports may not be generated.
 (-solver)=
 #### `--settings -solver=<solver spec>`
 
-This option sets the SMT solvers being used within the Prover.  By default, a
-portfolio of various different solvers is used.  It can be useful to specify
-only a subset of these to save on computation time.  In rare cases, solver
-specific options can improve performance as well.
-
-The `solver spec` can be a single solver (`-solver=z3`) or a list of solvers
-(`-solver=[cvc5,z3]`), where each such solver can be further modified.  For
-example, `cvc5` refers to the default configuration of `cvc5` whereas
-`cvc5:nonlin` is better for nonlinear problems.  Additional options can be set
-via `z3{randomSeed=17}`.
+By default, a portfolio of SMT solvers using various configurations is used within
+the Prover.  It can be useful to specify only a subset of these to save on
+computation time.  In rare cases, solver specific options can improve performance
+as well.  Setting `-solver=<solver spec>` filters the predefined portfolio to only
+use those configuration that match the given solver specification.
+
+The `solver spec` can be a single solver (`-solver=z3:def`), or a list of solver
+configurations (`-solver=[z3:def,cvc5:def]`), where each such solver can be
+further modified.  For example, `cvc5` refers to the set of pre-configured
```

Comment:
I would add here an explicit example of the CLI flag, i.e., "For example, `cvc5` (`-solver=cvc5`) refers to..." since the previous examples show only a specific configuration of solver.

Reply:
Do we actually have somewhere the whole list of available solver configurations? I am not sure the user can access it. So perhaps we should also add the list here?

Reply:
> Do we actually have somewhere the whole list of available solver configurations? I am not sure the user can access it. So perhaps we should also add the list here?

As before: IMHO we should not add lists of solver configurations here, as they will be outdated anyway. I'd rather establish a standard way to inspect this on a CVT executable.